CONTRIBUTING.md

```diff
@@ -52,4 +52,13 @@ accept your pull requests.
 
         npm run fix
 
+1. Run backend tests:
```

Comment:
Can you merge this with the previous test section and differentiate between "unit" and "system" tests (rather than backend).